### DISCUSSION TITLE
I have already filled in my academic email address but I am still asked to add my academic email address

### DISCUSSION THREAD

**Discussion by 20240608** [community/community#128605] [GitHub Education] [answered]
### Hi everyone,

Advice given after application for student certification is rejected：

                  Attendees of this school must verify and use their academic email address.

But I have added the academic email address given by the school and verified it before.
![屏幕截图 2024-06-15 103534](https://github.com/community/community/assets/160313716/b4f654b0-64b5-4f07-ae56-0c40680702b4)
Why do I need to add it? Why?

**Comment by samus-aran** [comment, marked as answer]
Hi @liudehua-creator 

Looks like that domain is not included in the selected school list. You will need to raise a support ticket if you feel the details we have are incorrect.

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
